sema/commons/glob/globbery.py

```
@@ -0,0 +1,125 @@
+
+
+class GlobMatchVisitor(ABC):
+    """Abstract interface for visiting paths matched by a glob."""
+
+    @abstractmethod
+    def visitExcluded(self, path: Path) -> None:
+        """
+        Called when a path is excluded by a glob.
+        @param path: The path that was excluded.
+        """
+        pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)